recipes-core/packagegroups/packagegroup-ni-base.bb

```diff
@@ -118,7 +119,6 @@ RDEPENDS_${PN} = "\
 	opkg-keyrings \
 	os-release \
 	run-postinsts \
-	start-stop-daemon \
```

Comment:
codesearch reveals that a lot of people use start-stop-daemon! Are you making a list of breaking changes that we can tell clients upfront? This should be added to that.

Reply:
Um, yeah. We're really not supporting that anymore? That will break a ton of stuff.

Reply:
The `start-stop-daemon` binary is still provided, just through the `dpkg-start-stop` package in oe-core, rather than the `start-stop-daemon` recipe in meta-oe. Meta-oe removed their recipe because the one in oe-core is newer (and from the same source).

So initscripts can continue to use `start-stop-daemon` in the same way. The only impact is if they have declared an IPK dependency on `start-stop-daemon`, it will need to be updated to `dpkg-start-stop`.